# goal
string row
string col
string priority
pkg_ros_iot_bridge/msgMqttSub order_data
---
# result
bool flag_success
string order_id
string city
string item
string priority
string dispatched_qty
string cost
string dispatched_status
string dispatched_time
---
# feedback
int8 percentage_complete

